1.8/administration/upgrading.md

```diff
@@ -136,7 +143,8 @@ Identify your Mesos leader node. This node should be the last master node that y
 
 1.  Validate the upgrade
 
-    Monitor the Mesos UI to verify that the upgraded node re-joins the DC/OS cluster and that tasks are reconciled (`http://<dcos_master>/mesos`).
+    - Verify that `curl http://<dcos_agent_private_ip>:5051/metrics/snapshot` has the metric `slave/registered` with a value of `1`.
+    - Monitor the Mesos UI to verify that the upgraded node rejoins the DC/OS cluster and that tasks are reconciled (`http://<dcos_master>/mesos`).
```

Comment:
For this for now I'd rather say Mesos UI, there are several behaviors of the DC/OS UI that work against what we want / need here currently.